python/coinbase-agentkit/coinbase_agentkit/agentkit.py

```diff
@@ -18,14 +18,14 @@ class AgentKitOptions(BaseModel):
 class AgentKit:
     """Main AgentKit class for managing wallet and action providers."""
 
-    def __init__(self, options: AgentKitOptions):
-        self.wallet_provider = options.wallet_provider
-        self.action_providers = options.action_providers or []
-
-    @classmethod
-    def from_options(cls, options: AgentKitOptions) -> "AgentKit":
-        """Create an AgentKit instance from options."""
-        return cls(options)
+    def __init__(self, config: AgentKitConfig):
+        self.wallet_provider = config.wallet_provider or CdpWalletProvider(
+            CdpWalletProviderConfig(
+                api_key_name=config.cdp_api_key_name,
+                api_key_private=config.cdp_api_key_private,
```

Comment:
```suggestion
                api_key_private_key=config.cdp_api_key_private_key,
```

can we make the required changes to have these postfixed with `_key` to align with TS